Replicator/Replicator.cc

```diff
@@ -172,6 +124,10 @@ namespace litecore { namespace repl {
             connection().start();
             // Now wait for _onConnect or _onClose...
 
+            if (!_options->isActive()) {
+                return;
```

Comment:
Probably should be assert?

Reply:
Well, passive replicator will also be started, but it will end here.